src/consensus/raft/raft.h

```diff
@@ -204,105 +178,104 @@ namespace raft
       recovery_max_index.reset();
     }
 
-    bool on_request(const kv::TxHistory::RequestCallbackArgs& args) override
-    {
-      return true;
-    }
-
-    void force_become_leader() override
+    void force_become_primary() override
     {
       // This is unsafe and should only be called when the node is certain
-      // there is no leader and no other node will attempt to force leadership.
-      if (leader_id != NoNode)
+      // there is no primary and no other node will attempt to force
+      // primaryship.
+      if (primary_id != NoNode)
         throw std::logic_error(
-          "Can't force leadership if there is already a leader");
+          "Can't force primaryship if there is already a primary");
 
       std::lock_guard<SpinLock> guard(lock);
       current_term += 2;
-      become_leader();
+      become_primary();
     }
 
-    void force_become_leader(Index index, Term term, Index commit_idx_)
+    void force_become_primary(
+      kv::SeqNo seqno, kv::View view, kv::SeqNo commit_seqno_)
     {
       // This is unsafe and should only be called when the node is certain
-      // there is no leader and no other node will attempt to force leadership.
-      if (leader_id != NoNode)
+      // there is no primary and no other node will attempt to force
+      // primaryship.
+      if (primary_id != NoNode)
         throw std::logic_error(
-          "Can't force leadership if there is already a leader");
+          "Can't force primaryship if there is already a primary");
 
       std::lock_guard<SpinLock> guard(lock);
-      current_term = term;
-      last_idx = index;
-      commit_idx = commit_idx_;
-      term_history.update(index, term);
+      current_term = view;
+      last_idx = seqno;
+      commit_idx = commit_seqno_;
+      term_history.update(seqno, view);
       current_term += 2;
-      become_leader();
+      become_primary();
     }
 
-    void force_become_leader(
-      Index index,
-      Term term,
+    void force_become_primary(
+      kv::SeqNo seqno,
+      kv::View view,
       const std::vector<Index>& terms,
-      Index commit_idx_) override
+      kv::SeqNo commit_seqno_) override
```

Comment:
As a follow on to the above, I think this should be  

```
void force_become_primary( 
      kv::SeqNo index,
      kv::View term,
      const std::vector<Index>& terms,
      kv::SeqNo commit_idx_) override
```
with the Consensus-general type names, but the Raft variable names. I see this as a self-documenting statement of the relationship between the abstract and the Raft-specific. (If someone asks "how do I get from an abstract `kv::View` to a Raft term? Is there a method that transforms them somewhere?", the answer is "no, they're the same thing", see this)